TwilioVerifySDK/TwilioSecurity/Sources/Storage/SecureStorage.swift

```diff
@@ -22,7 +22,7 @@ import Foundation
 ///:nodoc:
 public protocol SecureStorageProvider {
   func save(_ data: Data, withKey key: String, withServiceName service: String?) throws
-  func get(_ key: String) throws -> Data
+  func get(_ key: String, attempts: Int?) throws -> Data
```

Comment:
Do we need to define retries per storage type? For SecureStorageProvider we can do it and we need it according to the bug we found, and for AuthenticatedSecureStorage we can not do it in some cases

Reply:
Yeah absolutely, we could do something like that, for example we can define the verifyBiometrics using 1 attempt by default.